tests/merge_ntfs.rs

```diff
@@ -47,6 +52,36 @@ fn merge_collections_ok() {
     assert_eq!(collections.lines.len(), 6);
     assert_eq!(collections.routes.len(), 8);
     assert_eq!(collections.vehicle_journeys.len(), 8);
+    assert_eq!(collections.stop_time_headsigns.len(), 1);
```

Comment:
Don't you want to check to content also?

Reply:
done below